### PR TITLE
Tune Tx-Payment Parameters for Manta Runtime

### PR DESCRIPTION
**Pull request by ghzlatarev**
## Description
* Adds integration test
* Clean up unused code
* Tune `WeightToFeeCoeff` 

running congestion test output assuming price of manta of $0.36:
```
It will take 0.7008333 days to reach multiplier of 1x at a total cost of USD 3736.9966
Cost for 1 day in MANTA 392557645553367902640338 and in USD 141320.75
```

TODO:
- [ ] add congestion test warning comment to CI (same as in calamari)

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari`, `A-dolphin` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

**Comment by Garandor** [comment]
the manta tx part in this PR is obsoleted by #1111

File: runtime/integration-tests/src/integrations_mock/test_manta.rs [hunk 1]
```diff
@@ -198,6 +198,7 @@ fn reward_fees_to_block_author_and_treasury() {
             (ALICE.clone(), INITIAL_BALANCE),
             (BOB.clone(), INITIAL_BALANCE),
             (CHARLIE.clone(), INITIAL_BALANCE),
+            (Treasury::account_id(), INITIAL_BALANCE),
```
Comment (ghzlatarev) [inline runtime/integration-tests/src/integrations_mock/test_manta.rs:201]: wonder why this wasn't failing for the Calamari test

Comment (ferrell-code) [reply runtime/integration-tests/src/integrations_mock/test_manta.rs:201]: The existential deposit is lower 0.1 KMA compared to 1 MANTA.

**Comment by ghzlatarev** [comment]
looks good to me

**Review by Dengjianping — DISMISSED** [review]

**Review by ferrell-code — APPROVED** [review]

File: runtime/integration-tests/src/integrations_mock/test_manta.rs [hunk 2]
```diff
@@ -184,3 +190,77 @@ fn verify_manta_pallet_indices() {
...
+                author.clone(),
+            );
+
+            header.digest_mut().pop(); // pop the seal off.
+            System::initialize(&1, &Default::default(), header.digest());
+            assert_eq!(Authorship::author().unwrap(), author);
+
+            let call = RuntimeCall::Balances(pallet_balances::Call::transfer {
+                dest: sp_runtime::MultiAddress::Id(CHARLIE.clone()),
+                value: 10 * KMA,
```
Comment (Garandor) [inline runtime/integration-tests/src/integrations_mock/test_manta.rs:227]: this should be `* MANTA` not `* KMA`

File: runtime/integration-tests/src/integrations_mock/mod.rs [hunk 3]
```diff
@@ -73,7 +74,7 @@ use manta_primitives::types::{AccountId, Balance};
 use session_key_primitives::util::unchecked_account_id;
 use sp_core::sr25519::Public;
 
-pub const INITIAL_BALANCE: Balance = 1_000_000_000_000 * KMA;
+pub const INITIAL_BALANCE: Balance = 1_000_000_000_000_000_000_000_000_000;
```
Comment (Garandor) [inline runtime/integration-tests/src/integrations_mock/mod.rs:77]: use `* MANTA`

File: runtime/integration-tests/src/integrations_mock/mod.rs [hunk 4]
```diff
@@ -41,25 +42,25 @@ cfg_if::cfg_if! {
...
+            VotingPeriod, Preimage, NonPausablePallets, AllPalletsWithSystem
         };
         type RuntimeAssetConfig = calamari_runtime::assets_config::CalamariAssetConfig;
         type RuntimeConcreteFungibleLedger =
             calamari_runtime::assets_config::CalamariConcreteFungibleLedger;
     } else {
+        pub mod test_manta;
         use manta_runtime::{
-            assets_config::MantaConcreteFungibleLedger,
             currency::MANTA as KMA,
```
Comment (Garandor) [inline runtime/integration-tests/src/integrations_mock/mod.rs:54]: oh i see the code sets MANTA = KMA lol

pls make it `UNIT` and add the alias for both chains

Comment (ferrell-code) [reply runtime/integration-tests/src/integrations_mock/mod.rs:54]: Yeah I noticed this problem too it is due to the common test... we should fix this but I think it should be different PR. I'll log issue

Comment (ferrell-code) [reply runtime/integration-tests/src/integrations_mock/mod.rs:54]: https://github.com/Manta-Network/Manta/issues/1118 opened

**Review by Garandor — APPROVED** [review]
LGTM with #1118 as followup

**Pull request merged**